From c020527f8d734061253f8bfd1ca996cb774b35ba Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?J=C3=A1n=20Tomko?= <jtomko@redhat.com>
Date: Mon, 13 Aug 2018 13:37:53 +0200
Subject: [PATCH 01/16] Revert "src: Move DLOPEN_LIBS to libraries introducing
 the dependency"
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This reverts commit 5d40272ea67c74049600e120095d1b42287ed2d2.

Jansson cannot parse QEMU's quirky JSON.
Revert back to yajl.

https://bugzilla.redhat.com/show_bug.cgi?id=1614569

Signed-off-by: Ján Tomko <jtomko@redhat.com>
Reviewed-by: Daniel P. Berrangé <berrange@redhat.com>
---
 tools/Makefile.am | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/tools/Makefile.am b/tools/Makefile.am
index 1452d984a0..26c887649e 100644
--- a/tools/Makefile.am
+++ b/tools/Makefile.am
@@ -527,7 +527,8 @@ nss_libnss_libvirt_impl_la_CFLAGS = \
 
 nss_libnss_libvirt_impl_la_LIBADD = \
 	../gnulib/lib/libgnu.la \
-	../src/libvirt-nss.la
+	../src/libvirt-nss.la \
+	$(DLOPEN_LIBS)
 
 nss_libnss_libvirt_la_SOURCES =
 nss_libnss_libvirt_la_LDFLAGS = \
@@ -554,7 +555,8 @@ nss_libnss_libvirt_guest_impl_la_CFLAGS = \
 
 nss_libnss_libvirt_guest_impl_la_LIBADD = \
 	../gnulib/lib/libgnu.la \
-	../src/libvirt-nss.la
+	../src/libvirt-nss.la \
+	$(DLOPEN_LIBS)
 
 nss_libnss_libvirt_guest_la_SOURCES =
 nss_libnss_libvirt_guest_la_LDFLAGS = \
-- 
2.18.0

